app/controllers/application_controller.rb

```diff
@@ -122,4 +126,8 @@ def payment_link_url
     ENV['PAYMENT_LINK'] + show_reference_number
   end
   helper_method :payment_link_url
+
+  def delete_session
+    flash[:confirmation] = 'Session will expired'
```

Comment:
Do we have a design for this? I don't see anything in the trello ticket, might be worth confirming with design/content what to display here

Reply:
As discussed, values not used to display as toggle that that disappeared by next request